src/api/services/participantsService.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from 'zod';
+
+import { ParticipantSchema } from '../entities/Participant';
+import { ParticipantType } from '../entities/ParticipantType';
+import { createEmailService } from './emailService';
+import { EmailArgs } from './emailTypes';
+
+export const sendNewParticipantEmail = async (
+  newParticipant: z.infer<typeof ParticipantSchema>,
+  participantTypes: ParticipantType[]
+) => {
+  const emailService = createEmailService();
+  const requestor = newParticipant.users![0];
+  const templateData = {
+    participant: newParticipant.name,
+    participantType: participantTypes.map((pt) => pt.typeName).join(', '),
+    requestor: `${requestor.firstName} ${requestor.lastName}`,
+    requestorEmail: requestor.email,
+    participantLocation: requestor.location,
+    jobFunction: requestor.role,
+    link: 'http://localhost:3000/',
```

Comment:
can pull this from env ?

Reply:
Yes, we can, this link here is just for testing as we haven't confirmed what this link should be, should I just leave it blank for now?

Reply:
Yeah, or we can keep it as SSP_WEB_HOST, which is a sane default and it would at least lead them to home page